views/advanced_search.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Advanced search</h1>
+
+<div class="advanced-search-content">
+
+  <form>
+    <fieldset>
+      <legend>Chemical properties</legend>
+
+      <input id="advs-all" type="hidden" name="all" checked />
+
+      <input id="advs-peroxy" type="checkbox" name="peroxy" />
+      <label for="peroxy">Peroxy radical</label>
+
+      <input id="advs-submit" type="submit" value="Submit" />
+    </fieldset>
+  </form
+
+  <br/>
+
+  <% unless @species.nil? %>
```

Comment:
Can you add a count for the number of results found? Also visually separate results from the search boxes